lib/css/components/notices.less

```
@@ -1,252 +1,267 @@
         position: relative;
         width: 100%;
-        max-width: calc(var(--s-step) * 10);
-        margin: 0 auto;
     }
 
+    -webkit-transform: translate3d(0, var(--_no-x-offset), 0);
+    transform: translate3d(0, var(--_no-x-offset), 0);
+
     border-width: var(--su-static1) 0;
     inset: 0 0 auto 0;
     padding: var(--su12);
     position: fixed;
     width: 100%;
     z-index: calc(var(--zi-navigation-fixed) - 1); // Tuck below topbar
-    -webkit-transform: translate3d(0, var(--_no-x-offset), 0);
-    transform: translate3d(0, var(--_no-x-offset), 0);
 }
 
 // Notice
 .s-notice {
-    .construct-notice-component();
+    .construct-notice-component(s-notice);
+
     border-radius: var(--br-sm);
     border-width: var(--su-static1);
     padding: var(--su16);
 }
 
 // Toast
 .s-toast {
-    visibility: hidden;
-    position: fixed;
-    display: flex;
-    justify-content: center;
-    top: var(--su16);
-    left: var(--su8);
-    right: var(--su8);
-    opacity: 0;
-    z-index: calc(var(--zi-modals) + 1); // Toasts should appear above modals
-    transform: translate3d(0, -66px, 0);
-    transition: transform 100ms var(--te-smooth-slow) 0s, opacity 60ms var(--te-smooth-slow) 0ms, visibility 0s 150ms;
-    pointer-events: none;
-
+    @media (prefers-reduced-motion) {
+        transform: none !important;
+    }
     &[aria-hidden="false"] {
-        visibility: visible;
         opacity: 1;
         transform: translate3d(0, 0, 0);
         transition: visibility 0s 0s, opacity 100ms var(--te-smooth) 0s, transform 100ms var(--te-smooth) 0s;
-    }
-
-    @media (prefers-reduced-motion) {
-        transform: none;
+        visibility: visible;
     }
 
     .s-notice {
+        box-shadow: var(--bs-sm);
         max-width: 44rem;
-        width: 100%;
-        padding-top: var(--su8);
         padding-bottom: var(--su8);
-        box-shadow: var(--bs-sm);
+        padding-top: var(--su8);
         pointer-events: all;
+        width: 100%;
     }
+
+    display: flex;
+    justify-content: center;
+    left: var(--su8);
+    opacity: 0;
+    pointer-events: none;
+    position: fixed;
+    right: var(--su8);
+    top: var(--su16);
+    transform: translate3d(0, -66px, 0);
+    transition: transform 100ms var(--te-smooth-slow) 0s, opacity 60ms var(--te-smooth-slow) 0ms, visibility 0s 150ms;
+    visibility: hidden;
+    z-index: calc(var(--zi-modals) + 1); // Toasts should appear above modals
 }
+
+//  [1] When we use .s-banner, we need to adjust the padding-top on
+//      the body tag. This class correctly adjusts the body padding ONLY if
+//      the notice is one line. If it wraps to multiple lines, more classes or
+//      (ideally) JS will need to be used to determine the notice's height
+//      at the time of render. The padding value is determined like so:
+//      50px (top bar) + 44px (notice height) - 1px (bottom border)
+//      The borders subtraction are necessary to neatly tuck everything together.
```

[I'm not a monster](https://thoughtbot.com/blog/no-newline-at-end-of-file#history-lesson).

```suggestion
//      The borders subtraction are necessary to neatly tuck everything together.

```